requirements.txt

```
@@ -17,6 +17,10 @@ py2exe==0.11.1.0
 sphinx==3.4.1
 sphinx_rtd_theme
 
+# Requirements for type checking.
+# typing_extensions is incorporated in py3.8+, also available via mypy
+typing_extensions==4.2.0

```

Any particular reason for not using the latest version:
```suggestion
typing_extensions==4.3.0
```

---

No. I'll update that.